src/ol_infrastructure/substructure/keycloak/Pulumi.substructure.keycloak.CI.yaml

```diff
@@ -22,14 +22,14 @@ config:
   keycloak:okta_single_sign_on_service_url: https://dev-66940844.okta.com/app/dev-66940844_mitci_1/exkcshk0uw4gnJILX5d7/sso/saml
   keycloak:openid_clients:
   - client_info:
-      open-discusions: https://discussions-ci.odl.mit.edu
+      open-discusions: ["https://discussions-ci.odl.mit.edu"]
       open-local: "*"
     realm_name: olapps
   - client_info:
-      superset: https://bi-ci.ol.mit.edu
+      superset: ["https://bi-ci.ol.mit.edu", "superset_admin", "superset_alpha"]
```

Comment:
Rather than making this a mixed-use list, should it be a dictionary object? That seems like a more extensible and maintainable means of managing the information associated with clients. That could also allow for specifying additional client roles to be created, etc.